LDAR_Sim/src/methods/deployment/mobile_crew.py

```diff
@@ -162,7 +168,8 @@ def end_day(self, site_pool, itinerary):
             else:
                 self.choose_accommodation()
         elif len(itinerary) > 0:
-            self.update_schedule(itinerary[-1]['travel_home_mins'])
+            self.state['t'].current_date += timedelta(
+                minutes=int(itinerary[-1]['travel_home_mins']))
```

Comment:
This relates to mapping crews' trajectories PR. Since the timeseries is not passed to mobile_crew from crew, thus in order to record the last coordinates of the crew (for both on and off of scheduling), I suggest this function returns self.crew_lon and self.crew_lat. Just want to mention it here. We can discuss it in that PR.

Reply:
ohh you are right! This was in the code originally.